src/components/Tokens/TokenDetails/PriceChart.tsx

```diff
@@ -147,19 +150,25 @@ const crosshairDateOverhang = 80
 interface PriceChartProps {
   width: number
   height: number
+  token: Token
 }
 
-export function PriceChart({ width, height }: PriceChartProps) {
+export function PriceChart({ width, height, token }: PriceChartProps) {
   const [timePeriod, setTimePeriod] = useAtom(filterTimeAtom)
   const locale = useActiveLocale()
   const theme = useTheme()
 
+  const pricePoints: PricePoint[] = useTokenPriceQuery(token.address, timePeriod, 'ETHEREUM').filter(
```

Comment:
could you use the enum for ETHEREUM or whatever is exported to the function signature?

Reply:
There's a couple refactoring things like this to connect the front end to backend data types. Another example is like the backend uses a DURATION variable, but we have a separate TimePeriod enum on the front end, and these things can probably be merged. But if I did all these tweaks this PR would get bigger. I figured it was better to keep the scope of this PR smaller, what do you think?

Reply:
Will do!